app/talk/board-preview.cjsx

```diff
@@ -31,16 +31,21 @@ module?.exports = React.createClass
 
         <PromiseRenderer promise={talkClient.type('discussions').get({board_id: @props.data.id}).index(0)}>{(discussion) =>
           if discussion?
-            <div className="talk-discussion-link">
-              <PromiseRenderer promise={apiClient.type('users').get(discussion.user_id.toString())}>{(user) =>
-                <Link className="user-profile-link" to="user-profile" params={name: discussion.user_login}>
-                  <Avatar user={user} />{' '}{discussion.user_display_name}
-                </Link>
-              }</PromiseRenderer>{' '}
-
-              <Link to="#{@projectPrefix()}talk-discussion" params={merge({}, {board: discussion.board_id, discussion: discussion.id}, @props.params)}>{discussion.title}</Link>{' '}
-              <span>{timeAgo(discussion.updated_at)}</span>
-            </div>
+            comments = discussion.links.comments
```

Comment:
Any way we can get the last comment without accessing `links` directly? No big deal if we can't, but `links` can be expressed lots of different ways in JSON API, I hate depending on our implementation to never change.